src/wallet/fees.cpp

```diff
@@ -20,6 +20,22 @@ CAmount GetRequiredFee(unsigned int nTxBytes)
 
 
 CAmount GetMinimumFee(unsigned int nTxBytes, const CCoinControl& coin_control, const CTxMemPool& pool, const CBlockPolicyEstimator& estimator, FeeCalculation *feeCalc)
+{
+    CAmount fee_needed = GetMinimumFeeRate(coin_control, pool, estimator, feeCalc).GetFee(nTxBytes);
+    // Aalways obey the maximum
```

Comment:
typo: Aalways

Note to self, got up to here.

Reply:
Fixed